src/c_kzg_4844.c

```diff
@@ -53,6 +54,21 @@
 static const char *FIAT_SHAMIR_PROTOCOL_DOMAIN = "FSBLOBVERIFY_V1_";
 static const char *RANDOM_CHALLENGE_KZG_BATCH_DOMAIN = "RCKZGBATCH___V1_";
 
+/** The number of bytes in a protocol domain. */
+static const size_t BYTES_PER_DOMAIN = 16;
+
+/* Input size to the Fiat-Shamir challenge computation. */
+static const size_t CHALLENGE_INPUT_SIZE = 32 + BYTES_PER_BLOB + 48;
+
+/** The number of bytes in a g1 point. */
+static const size_t BYTES_PER_G1 = 48;
+
+/** The number of bytes in a g2 point. */
+static const size_t BYTES_PER_G2 = 96;
+
+/** The number of g2 points in a trusted setup. */
+static const size_t NUM_G2_POINTS = 65;
```

Comment:
Perhaps `TRUSTED_SETUP_G2_POINTS_LEN` or something to make it more specific? Or perhaps move this constant down in the trusted setup section?

(I'm not doing a github code suggestion because I can't change both places i think)